### DISCUSSION TITLE
Adding Comparison & Viewing Exif Feature

### DISCUSSION THREAD

**Discussion by gaduaxe** [d2phap/ImageGlass#839] [General] [answered]
Hi Bros

It's really nice if you can add Comparison (viewing side by side: zooming, moving....) of 2/3/4...multiple images
also ImageGlass can show details EXIF of Image if needed

Tks for reading

**Comment by d2phap** [comment]
Hi @gaduaxe,

Viewing Exif feature is planned to add in the next release: https://github.com/d2phap/ImageGlass/issues/336

**Comment by d2phap** [comment, marked as answer]
Exif viewer tool is ready in IG Moon https://imageglass.org/moon
